ci-build.sh

```diff
@@ -166,11 +173,20 @@ if [ $TEMP_POSTGRES -eq 0 ] ; then
     done
 fi
 
-export ALL_VERSIONS=1
-export NUM_PARTITIONS=$((NPROCS*2))
-export RUN_PARTITIONS
 ulimit -n 256
-time make check
+
+if [ $CHECK_TEST_TX_META -eq 1 ] ; then
+    time src/stellar-core test \
```

Comment:
This doesn't look right: you should be able to run in parallel.

All you need to do is override
`STELLAR_CORE_TEST_PARAMS` and `TEST_SPEC` before calling `make check`

Reply:
Ah, I didn't know those existed for setting parameters for make check. Fixed.